File: Modul-4/Soal 4/tempCodeRunnerFile.c
#include <stdio.h>
int main(void){
    float  r, x, y;

    while(1){
        printf("Pilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan Pilihan =\t");
        scanf("%f", &r);
        
        if(r>5 || r<1){printf("Input anda salah, silahkan coba lagi"); continue; }
        else if(r==5) {printf("Terimakasih, telah menggunakan kalkulator pacarnya rafayel"); break;}
    printf("Masukkan nilai pertama : ");
    scanf("%f", &x);
    printf("Masukkan nilai kedua : ");
    scanf("%f", &y);
    if(r==1){printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f\n",x,y, x+y);}
    else if(r==2){printf("Hasil Pengurangan antara %.2f dengan %.2f adalah %.2f\n",x,y, x-y);}
    else if(r==3){printf("Hasil Perkalian antara %.2f dengan %.2f adalah %.2f\n",x,y, x*y);}
    else if(r==4){if(y!=0){printf("Hasil pembagian antar %.2f dengan %.2f adalah %.2f\n",x,y, x/y); }
    else{printf("Hasil tidak terdefinisi\n");}
    }
}
}

File: Modul-4/Soal 4/tempCodeRunnerFile.py
def kalkulator():
    name = input("Masukkan nama Anda : ")

    while True:
        print("Pilih program")
        print("1. Penjumlahan")
        print("2. Pengurangan")
        print("3. Perkalian")
        print("4. Perkalian")
        print("5. Exit")

        lads = int(input("Masukkan Pilihan :"))
        if lads == 5:
            print(f"Terimakasih, telah menggunakan kalkulator {name}")
            break
        elif lads in[1, 2, 3, 4]:
            rafa = float(input("Masukkan nilai pertama :"))
            caleb = float(input("Masukkan nilai kedua :"))

            if lads == 1:
                rey = rafa + caleb
                print(f"Hasil penjumlahan antara {rafa:.2f} dengan {caleb:.2f} adalah {rey:.2f} \n")
            elif lads == 2:
                rey = rafa - caleb
                print(f"Hasil Pengurangan antara {rafa:.2f} dengan {caleb:.2f} adalah {rey:.2f} \n")
            elif lads == 3:
                rey = rafa * caleb
                print(f"Hasil Perkalian antara {rafa:.2f} dengan {caleb:.2f} adalah {rey:.2f} \n")
            elif lads == 4:
                rey = rafa / caleb
                print(f"Hasil Pembagian antara {rafa:.2f} dengan {caleb:.2f} adalah {rey:.2f} \n")
        else:
            print("Input anda salah, silahkan coba lagi \n")
kalkulator()